index.html

```diff
@@ -29,8 +28,11 @@ <h1 class="main-heading">I'm Celyne.<br> Glad to see you!</h1>
           <li><a href="#" class="icons"><img src="/images/assets/github.png"></a></li>
           <li><a href="#" class="icons"><img src="/images/assets/hi 5.png"></a></li>
         </ul><br>
+        <button type="button">Get my resume</button> 
+  
         </div>
       </section>
```

Comment:
- Please make sure you add the proper padding and spacing for the main section here as shown in the design
      ![ApplicationFrameHost_TgYJFZFGqZ](https://user-images.githubusercontent.com/104968236/227521949-72d3da38-49d1-4c03-8578-9b30c4e60a25.png)

- I've highlighted the left and right padding you seem to be missing and kindly add `100vh` as the height of this section as per the [Figma design](https://github.com/musangamfure/portfolio/pull/1/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R40-R62).
     ![Inkedfirefox_idvdmwbhjL](https://user-images.githubusercontent.com/104968236/227522684-40e24671-882b-4998-9234-ba7766b61f83.jpg)